.github/workflows/ci.yml

```diff
@@ -52,11 +52,12 @@ jobs:
           flutter pub get
           cd ../..
           cd packages/flutter
+          echo "publish_to: none" >> pubspec.yaml
```

Comment:
What does this do?

Reply:
flutter analyze failes if the dart package is added using path since it would not work when publishing the package. This trick tells the analyzer we don't want to publish the package. (What is actually not true.)

Reply:
Cool!